### PR TITLE
online-bookstore: Connect with API

### PR DESCRIPTION
**Pull request by BertrandConxy**
# online-bookstore: Connect with API

Hello ✋ 

I would like to share with you my work. In this branch, I have done the following:
- Added redux-thunk package in order to be able to perform async actions.
- Added thunk in the middleware
- Refactored the add-book action and the remove-book action
- Add new action which is load books.

After all, I checked if all features are working and they are. Looking forward to your review.

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -1,50 +1,56 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (rustam-tolipov) [inline src/components/BookItem.js:1]: - [x] [OPTIONAL] **GOOD JOB 👏👌**. One thing that its a little hard to see the bottom **ADD NEW BOOK** part, so please kindly add some space to the bottom 

image

**Review by rustam-tolipov — APPROVED** [review]
## HELLO @BertrandConxy 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/BookItem.js (hunk 1 above)
Comment (BertrandConxy) [reply src/components/BookItem.js:1]: Yeah. sure. I am going to implement that. Thank you very much @Rustamxon7

**Comment by BertrandConxy** [comment]
Thank you very much @Rustamxon7

File: src/components/BookItem.js (hunk 1 above)
Comment (rustam-tolipov) [reply src/components/BookItem.js:1]: You are welcome 🙌

**Pull request merged**